### PR TITLE
#24 groups

### PR DESCRIPTION
**Pull request by nicknapoli82**
Getting close to some final touches. Need to work on adding Posts to the Groups components now.
- User can update the group if they are an admin
- Some small touch ups (more needed I think)

### CODE REVIEW

File: client/components/Group/GroupContext.js [hunk 1]
```diff
@@ -3,31 +3,39 @@ import { Container, Grid, Item, List, Image, Segment, Button} from 'semantic-ui-
...
 import { Link } from 'react-router-dom';
 
+import UpdateGroupForm from './UpdateGroupForm';
+
 class GroupContext extends React.Component {
   render() {
-    const { context, history, groupId, isMember, isAdmin } = this.props;
+    const { context, history, groupId, isMember, isAdmin, adminRemoveMember } = this.props;
     if(this.props.groupDetailed[context] === undefined || context === undefined) return null;
+    console.log('context ', context);
```
Comment (justin-cook-developer) [inline client/components/Group/GroupContext.js]: Do you want to keep this log?

File: client/reducers/groupReducer.js [hunk 2]
```diff
@@ -41,6 +41,15 @@ export const leaveGroup = () => (dispatch, getState) => {
     .catch(e => console.log("I suck at math"));
 };
 
+export const adminRemoveMember = (userId, groupId) => (dispatch, getState) => {
+  console.log(userId, groupId);
```
Comment (justin-cook-developer) [inline client/reducers/groupReducer.js]: Log?

File: server/apiRoutes/group.js [hunk 3]
```diff
@@ -219,6 +233,7 @@ router.post('/addmember', async (req, res, next) => {
 // This is for an admin/owner to remove a user from a group
 router.delete('/removemember', async (req, res, next) => {
   try {
+    console.log(req.body);
```
Comment (justin-cook-developer) [inline server/apiRoutes/group.js]: Log?

**Review by justin-cook-developer — CHANGES_REQUESTED** [review]

File: client/components/Group/GroupContext.js (hunk 1 above)
Comment (nicknapoli82) [reply client/components/Group/GroupContext.js]: Oh... Damn, I thought I killed those.

**Review by justin-cook-developer — APPROVED** [review]

**Pull request merged**